### DISCUSSION TITLE
账号被误封

### DISCUSSION THREAD

**Discussion by 4771566HermosaCronin** [BurnHair/docs#134] [技术支持]
今天登录提示用户已被封禁，但我没有重复注册账号也没有与他人共享key
用户名：4771566HermosaCronin

**Comment by BennyThink** [comment]
你与用户 shiyanbeichen 的注册与请求IP地址均相同，因此被判定为重复注册。

**Reply by 4771566HermosaCronin** [reply]
我完全没这个账号 只此一个账号

**Reply by 4771566HermosaCronin** [reply]
我登录注册均使用了机场，也曾用render上部署的new-api调用过
不知道与这些是否有关系

**Reply by BennyThink** [reply]
你与这个用户有相同的上下文。

**Reply by 4771566HermosaCronin** [reply]
我最近没进行调用，具体什么时候的能发来看看吗

**Reply by 4771566HermosaCronin** [reply]
> 你与用户 shiyanbeichen 的注册与请求IP地址均相同，因此被判定为重复注册。

麻烦给出相同的上下文
我从没注册过其他账号也没有分享过key
